src/components/MessageBar/index.tsx

```diff
@@ -0,0 +1,42 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { NestableBaseComponentProps } from '../../util/BaseComponent/props';
+import { Enum } from 'typescript-string-enums';
+import './message-bar.css';
+
+export const MessageBarType = Enum({
+  INFO: 'info',
+  WARNING: 'warning',
+  ERROR: 'error',
+});
+export type MessageBarType = Enum<typeof MessageBarType>;
+
+export interface MessageBarProps extends NestableBaseComponentProps {
+  ariaLabel: string;
+  type?: MessageBarType;
+  actions?: JSX.Element;
+  children: JSX.Element | string;
+}
+
+const getActions = (props: MessageBarProps) => {
+  if (!props.actions) {
+    return null;
+  }
+  return (<div className="y-message-bar--actions">{props.actions}</div>);
+};
+
+const MessageBar: React.StatelessComponent<MessageBarProps> = (props) => {
+  const actions = getActions(props);
+  return (
+    <div className={`y-message-bar y-message-bar__type-${props.type}`} aria-label={props.ariaLabel}>
```

Comment:
All our components can take a `className`. You can do something like this: https://github.com/Microsoft/YamUI/blob/958d59a11d6035debf675379c434077c1d2c28f4/src/components/Callout/callout.tsx#L135

Reply:
Alright - is that documented?

Reply:
Feel free to document it. I picked it up by looking at all the existing components, and the tests for the existing components. They all check that `className` can be passed.